src/App.js

```diff
@@ -1,12 +1,49 @@
-import React from 'react';
-import './App.css';
+import { useState, useEffect } from "react";
+import axios from "axios";
+import "./App.css";
+import PokemonCard from "./Components/PokemonCard";
```

Comment:
De `components` map mag je veranderen in een kleine letter (in zowel de map-naam als de import)